gui/tray_applet/razer_tray_applet.py

```diff
@@ -430,7 +430,7 @@ def apply_startup_settings(self):
             if not start_brightness == 0:
                 self.daemon.set_brightness(start_brightness)
 
-            if start_macro:
+            if start_macro == 'true':
```

Comment:
Instead of == 'true', couldn't you leave in the False defaults but remove the bool(), as a string will evaluate to True anyway?